tests/async/test_browsertype_connect.py

```diff
@@ -241,3 +243,54 @@ async def handle_request(route: Route) -> None:
     assert await response.json() == {"foo": "bar"}
 
     remote.kill()
+
+@pytest.mark.only_browser("chromium")
+async def test_should_upload_large_file(
+    browser_type: BrowserType,
+    launch_server: Callable[[], RemoteServer],
+    playwright: Playwright,
+    server: Server,
+    tmp_path):
+    remote = launch_server()
+
+    browser = await browser_type.connect(remote.ws_endpoint)
+    context = await browser.new_context()
+    page = await context.new_page()
+
+    await page.goto(server.PREFIX + "/input/fileupload.html")
+    large_file_path = tmp_path / "200MB.zip"
+    data = b'A' * 1024
+    with large_file_path.open("wb") as f:
+        for i in range(0, 200 * 1024 * 1024, len(data)):
```

Comment:
nit: lets have it like we have upstream. Instead of 204k for loop iterations only 50k!

Reply:
I think this is equivalent to upstream; the step param is `len(data)`